packages/iov-bns/src/types.ts

```diff
@@ -93,10 +93,33 @@ export enum ProposalStatus {
   Withdrawn,
 }
 
-/** Union type for possible options */
-export type ProposalOption = string;
+export enum VoteOption {
+  Yes,
+  No,
+  Abstain,
+}
+
+export interface TallyResult {
+  readonly totalYes: number;
+  readonly totalNo: number;
+  readonly totalAbstain: number;
+  readonly totalElectorateWeight: number;
+}
+
+export interface CreateTextResolution {
```

Comment:
I find the word `Create` a bit confusing here.

Reply:
I think the naming issue is in "option". This field holds an action to be performed when the proposal is accepted. All of those action [got verb-prefixed recently](https://github.com/iov-one/weave/issues/738). My personal intuition was to use "publish text resolution", but then I used the [new name from weave 0.17](https://htmlpreview.github.io/?https://github.com/iov-one/weave/blob/v0.17.0/docs/proto/index.html#gov.CreateTextResolutionMsg).

Thoughts?

Reply:
Yeah, "action" would be much clearer. Could we make it `CreateTextResolutionAction`?

Reply:
In https://github.com/iov-one/iov-core/pull/1096/commits/e7fffb539e95e800c67a1b57585dca05fd2dba4f I renamed the "option"s to "action"s. I think we don't need it as a suffic for every single type in `ProposalAction` then.